demo/admin/src/products/future/ProductsGrid.cometGen.ts

```diff
@@ -11,6 +11,14 @@ export const ProductsGrid: GridConfig<GQLProduct> = {
     toolbarActionProp: true,
     rowActionProp: true,
     excelExport: true,
+    moreActions: {
+        overallActions: [
+            {
+                name: "publish",
```

Comment:
This is only used for the key of the component in the array (for now). Should I rename it to key?

Reply:
why do you need the key?

Reply:
Because overallActions in CrudMoreActionsMenu expects an array, and React expects a key when using React components in an array.